app/controllers/idv/doc_auth_controller.rb

```diff
@@ -19,6 +19,13 @@ class DocAuthController < ApplicationController
       analytics_id: Analytics::DOC_AUTH,
     }.freeze
 
+    def return_to_sp
+      redirect_to return_to_sp_failure_to_proof_url(
+        step: next_step,
+        **params.permit(:location).to_h.symbolize_keys,
```

Comment:
ditto for the one key:

```suggestion
        location: params[:location],
```

Reply:
Updated in 82d7f70.